src/pages/projects/ProjectsFilterAndSort.tsx

```diff
@@ -87,6 +91,11 @@ export default function ProjectsFilterAndSort({
               checked={showArchived}
               onChange={setShowArchived}
             />
+            <FilterCheckboxItem
+              label={t`Reverse`}
```

Comment:
**question** hmm, i wonder if this should be in a different "Sort order" dropdown.

WDYT @strath-m ?